src/Database/Connection.php

```diff
@@ -107,7 +107,14 @@ class Connection implements ConnectionInterface
     /**
      * Constructor.
      *
-     * @param array $config configuration for connecting to database
+     * @param array $config Configuration array.
+     * ### Available options:
```

Comment:
I would prefer those going on top before params start, as we usually do in other places.